Model/DutyCalculator.php

```diff
@@ -244,6 +249,20 @@ public function handleCaseWhereDutyIsAlreadyRetrieved($apply, $quote, $address)
         $this->response->setDuty($this->checkoutSession->getReachDuty());
     }
 
+    /** gets duty in base currency from display currency
+     * @param float $duty
+     * returns float
+     */
+    public function getBaseDuty($duty)
+    {
+        //it may look at stale value???
```

Comment:
could you explain a bit more about what this comment means? I don't have enough context to know if something should be done about this, if this is a bug, etc

Reply:
The note was pretty much like talking to myself/set as a hint that there might be a window of uncertainty /invalid rate based on when rate was fetched and when it is applied to get base currency equivalent from display currency. Not sure what would be the frequency of such occurrences and corresponding implications. I think you mentioned this briefly when you were demonstrating the payment section in front of us (why fetching every six hour is reasonable ... and so on)  Lizzy. Thanks! Do you want me to remove this comment?

Reply:
ohh that makes sense, I could see either: 

- expanding the comment to include more of what you were thinking about
- removing the comment

it's a small thing, so only if it makes sense to you too

Reply:
Removed. Thanks! commit 971e7cc32fde044bdeb84a06f3d5308baf9713cb

Reply:
thank you!